katas/content/complex_arithmetic/Common.qs

```diff
@@ -11,15 +11,17 @@ namespace Kata.Verification {
     }
 
     function ComplexAsString(x : Complex) : String {
-        if x.Imag < 0.0 {
-            $"{x.Real} - {AbsD(x.Imag)}i"
+        let precision = 3;
+        if x::Imag < 0.0 {
```

Comment:
I realized what was causing the conflict - an earlier PR that updated the syntax for accessing UDT fields from :: to . (you can see it later in this file, for example, ComplexEqual and ComplexPolarEqual use .)

This change reverts it in part of the file; can you please update your change to the new syntax for consistency?

Reply:
Got it. I have updated the file now.